docs/qsimcirq_tutorial.ipynb

```diff
@@ -0,0 +1,636 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "qsimcirq_tutorial.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vAx5g2rPnlzt",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Getting started with qsimcirq\n",
+        "\n",
+        "The qsim library provides a Python interface to Cirq in the **qsimcirq** PyPI package."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "XcAhsIQdivnp",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "%%shell\n",
+        "pip install cirq==0.8.0 --quiet\n",
+        "pip install qsimcirq==0.3.0 --quiet"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FXytmQlBoeTT",
+        "colab_type": "text"
+      },
+      "source": [
+        "Simulating Cirq circuits with qsim is easy: just define the circuit as you normally would, then create a **QSimSimulator** to perform the simulation. This object implements Cirq's [simulator.py](https://github.com/quantumlib/Cirq/blob/master/cirq/sim/simulator.py) interfaces, so you can drop it in anywhere the basic Cirq simulator is used.\n",
+        "\n",
+        "## Full state-vector simulation\n",
+        "\n",
+        "qsim is optimized for computing the final state vector of a circuit. Try it by running the example below."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "jd8DaJMbjkCM",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 139
+        },
+        "outputId": "a1b4abf1-3405-4d93-f514-eeb43cf06329"
+      },
+      "source": [
+        "import cirq\n",
+        "import qsimcirq\n",
+        "\n",
+        "states = ['00', '01', '10', '11']\n",
+        "\n",
+        "# Define qubits and a short circuit.\n",
+        "q0, q1 = cirq.LineQubit.range(2)\n",
+        "circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))\n",
```

Comment:
I think it would be useful to print out the circuit diagram for people to look at